ipywidgets/widgets/interaction.py

```diff
@@ -79,10 +80,26 @@ def _widget_abbrev_single_value(o):
 
 def _widget_abbrev(o):
     """Make widgets from abbreviations: single values, lists or tuples."""
-    if isinstance(o, (list, tuple)):
-        if o and all(isinstance(x, string_types) for x in o):
+    if isinstance(o, list):
+        # --------------------------------------------------------------------
+        # Handle deprecated behavior of using lists of length 2 or 3 in place
+        # of tuples to specify slider widget attributes. This will be removed
+        # in ipywidgets 6.0.
+        if len(o) in [2, 3] and all(isinstance(x, Real) for x in o):
```

Comment:
Given the comments below about tuples vs. lists in the docs, I guess `1 in [2,3]` is more pythonic, especially if it is optimized the same.